docker/centos-graal-maven-s2i/s2i/run

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+# This POC script is (intentionally) overly trivial.
+# a TODO future PRODUCTION version of this would probably want tomake it possible to
+# e.g. pass additional parameters to the executable, as does fabric8io-images/s2i's run script.
+
+/quarkus.io/*-runner -Dquarkus.http.host=0.0.0.0
```

Comment:
For the record, there are discussions about the 0.0.0.0 thingy.

Reply:
Yup, so I gathered.. if/when that happens, someone needs to drop this from a whole lot of quickstarts and docs and scripts and all - including this. For now, it's required though.